README.rst

```diff
@@ -28,3 +28,8 @@ I want to do an automated installation!
 ---------------------------------------
 
 Kickstart has you covered! Check out the docs at: https://pykickstart.readthedocs.io
+
+I want to hear more about the project!
+--------------------------------------
+
+Follow our blog website at: https://rhinstaller.wordpress.com
```

Comment:
Subjective... To me the "hear" phrasing feels static and misleading, as in "more detailed information right now". That's in contrast to what you are trying to convey: an ongoing process, where the reader gets further information as it is made available. So I think the keywords could be "keep in touch", "follow", "subscribe", "updates" and along these lines, to suggest continuity and suppress the "more details" outcome.

You could probably also use a continuous tense like "I want to keep hearing from you about changes". But that doesn't go well with the actual words I suggested above.

Reply:
I've tried to make the change. Not sure if correctly. If not we can discuss that in the office on Monday.